infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutContextData.kt

```diff
@@ -46,13 +44,14 @@ sealed class LayoutAsset<T : LayoutAsset<T>>(contextData: LayoutContextData<T>)
 
 sealed class LayoutContextData<T> : LayoutContextAware<T> {
 
-    @get:JsonIgnore abstract val rowId: IntId<T>?
+    @get:JsonIgnore abstract
+    val rowId: LayoutRowId<T>?
```

Comment:
Näitä tyyppejä vaihtaessa tuntui vahvasti että tämä on hyvä suunta.

Reply:
Muutoksena vaikuttaa kyllä mielekkäältä 👍 

(Tässä muuten hieman hassusti tuo abstract keywordi jäi ylemmälle riville)